src/connector/src/sink/kafka.rs

```diff
@@ -71,6 +72,25 @@ const fn _default_force_append_only() -> bool {
     false
 }
 
+#[derive(Debug, Clone, PartialEq, Display, Serialize, Deserialize, EnumString)]
+enum CompressionCodec {
+    #[serde(rename = "none")]
```

Comment:
not sure if we need serde here, appreciate some guidance

Reply:
I guess we don't. `#[serde_as(as = "Option<DisplayFromStr>")]` will delegate the implementation to `FromStr` and `Display`.

Reply:
updated